packages/plugins/plugin-sheet/src/components/RangeList/RangeList.tsx

```diff
@@ -2,37 +2,51 @@
 // Copyright 2024 DXOS.org
 //
 
-import React from 'react';
+import React, { useCallback } from 'react';
 
 import { S } from '@dxos/echo-schema';
+import { useTranslation } from '@dxos/react-ui';
 import { List } from '@dxos/react-ui-list';
 import { ghostHover } from '@dxos/react-ui-theme';
 
 import { rangeFromIndex, rangeToA1Notation } from '../../defs';
+import { SHEET_PLUGIN } from '../../meta';
 import { Range, type SheetType } from '../../types';
 
 export type RangeListProps = {
   sheet: SheetType;
-  onSelect?: (range: Range) => void;
-  onDelete?: (range: Range) => void;
 };
 
-export const RangeList = ({ sheet, onSelect, onDelete }: RangeListProps) => {
+export const RangeList = ({ sheet }: RangeListProps) => {
+  const { t } = useTranslation(SHEET_PLUGIN);
+  const handleSelectRange = (range: Range) => {};
```

Comment:
TODO to back-prop this to the grid to show the currently active range? I think this is like comments select.

Reply:
Good idea, will leave a comment and make an issue.